api/db-cdr/changelog-data/db.changelog-5-icd10-2.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="brianfreeman" id="changelog-5-icd10-2" runAlways="true" context="cloud">
+
+    <validCheckSum>ANY</validCheckSum>
+
+    <loadData tableName="criteria" file="csv/icd10_criteria-2.csv" encoding="UTF-8" quotchar='"'>
```

Comment:
What's the difference between each of these CSVs? Are we breaking them up just to make it possible for Liquibase to import them?

You may want to add a comment.

Reply:
just wanted to make the files smaller cause i did get an out of memory exception once while debugging these changes.  Added comments.